code/WorkInProgress/mantaObjects.dm

```diff
@@ -716,6 +716,110 @@ var/obj/manta_speed_lever/mantaLever = null
 	stamina_damage = 15
 	stamina_cost = 8
 	stamina_crit_chance = 10
+	max_stack = 5
+	item_state = "cone_1"
+	wear_state = "cone_hat_1"
+
+	setupProperties()
+		..()
+		setProperty("coldprot", 0) // it has a hole on top, after all
+		setProperty("heatprot", 0)
+		setProperty("meleeprot_head", 2)
+
+	before_stack(atom/movable/O as obj, mob/user as mob)
+		user.visible_message("<span class='notice'>[user] begins gathering up [src]\s!</span>")
+
+	after_stack(atom/movable/O as obj, mob/user as mob, var/added)
+		UpdateStackAppearance()
+		boutput(user, "<span class='notice'>You finish gathering up [src]\s.</span>")
+
+	attack_hand(mob/user)
+		if((user.r_hand == src || user.l_hand == src) && src.amount > 1)
+
+			if (!in_interact_range(src, user)) //no walking away
+				return
+
+			var/obj/item/clothing/head/constructioncone/new_stack = split_stack(1)
+			if (!istype(new_stack))
+				boutput(user, "<span class='alert'>Invalid entry, try again.</span>")
+				return
+			user.put_in_hand_or_drop(new_stack)
+			new_stack.add_fingerprint(user)
+			boutput(user, "<span class='notice'>You take 1 cone from the stack, leaving [src.amount] cones behind.</span>")
+		else
+			..(user)
+
+	check_valid_stack(atom/movable/O as obj)
+		if (!istype(O, /obj/item/clothing/head/constructioncone))
+			return 0
+		return 1
```

Comment:
this should be unnecessary since it's roughly what the default implementation does?